include/RED4ext/CommandLine-inl.hpp

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#ifdef RED4EXT_STATIC_LIB
+#include <RED4ext/CommandLine.hpp>
+#endif
+
+#include <RED4ext/Detail/AddressHashes.hpp>
+#include <RED4ext/Relocation.hpp>
+#include <RED4ext/StringView.hpp>
+
+RED4EXT_INLINE RED4ext::CommandLine* RED4ext::CommandLine::Get() noexcept
+{
+    // NOTE: GetAddr is actually necessary here
+
+    static CommandLine* ptr{
+        RED4ext::UniversalRelocPtr<RED4ext::CommandLine>(RED4ext::Detail::AddressHashes::CommandLine).GetAddr()};
+    return ptr;
+}
+
+RED4EXT_INLINE bool RED4ext::CommandLine::HasLaunchParameter(RED4ext::StringView aParam) noexcept
+{
+    using func_t =
+        bool (*)(RED4ext::CommandLine* aThis, StringView aParam, std::uint32_t* aOptionIndex, std::uint64_t** aUnk2);
```

Comment:
It's probably `StringView* aParam`?

Reply:
StringViews tend to be passed on the stack, I would argue this may be right but would have to check disasm as for the other thing. Would make sense though for it to be like this I believe.

Reply:
discussed on discord, change please